README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status](https://travis-ci.com/qbicsoftware/omero-client-lib.svg?branch=development)](https://travis-ci.com/qbicsoftware/omero-client-lib)[![Code Coverage]( https://codecov.io/gh/qbicsoftware/omero-client-lib/branch/development/graph/badge.svg)](https://codecov.io/gh/qbicsoftware/omero-client-lib)
 
-OMERO client library, version 1.0.0-SNAPSHOT - Java-based library to connect to the OMERO server
+OMERO client library, version 1.1.0 - Java-based library to connect to the OMERO server
```

Comment:
```suggestion
OMERO client library, version 1.2.0-SNAPSHOT - Java-based library to connect to the OMERO server
```

Reply:
Maybe merge it locally into development. That does not change the release/1.1.0 branch